src/lib/routes/client-api/feature.ts

```diff
@@ -118,7 +128,22 @@ export default class FeatureController extends Controller {
                 featureQuery,
             );
         }
-        res.json({ version, features, query: featureQuery });
+
+        const response = {
+            version,
+            features,
+            query: featureQuery,
+        };
+
+        if (this.useGlobalSegments) {
+            const segments = await this.segmentService.getActive();
+            res.json({
+                ...response,
+                segments,
```

Comment:
Could probably remove `SegmentsController` and `enableSegmentsClientApi`, this seems better.

Reply:
Oh neat, if we don't need them for anything, can I delete them and include them in this PR?

Reply:
Yeah, that was the other approach where we'd fetch segments from their own endpoint.

Would need a corresponding enterprise PR since `enableSegmentsClientApi` is used there as well.

Reply:
Cool, I'm a huge fan of deleting code! I think let me do that in a separate pair of PRs though